.github/workflows/ci.yml

```diff
@@ -54,3 +54,34 @@ jobs:
       with:
         directory: ./coverage/
       timeout-minutes: 3
+
+  release:
+    runs-on: ubuntu-latest
+    # if: github.event_name == 'push'
```

Comment:
Will this be uncommented in the future?  Should this be restricted to pushes to `master` like https://github.com/semantic-release/semantic-release/blob/master/docs/recipes/github-actions.md#githubworkflowsreleaseyml-configuration-for-node-projects?  When a push occurs, does the actual release only happen if the title of the commit is `Release: ...`?

Reply:
> Will this be uncommented in the future?

Yes, I forgot to uncomment it. I just wanted to check it runs: https://github.com/KaTeX/KaTeX/runs/2089146698.

> Should this be restricted to pushes to `master` like https://github.com/semantic-release/semantic-release/blob/master/docs/recipes/github-actions.md#githubworkflowsreleaseyml-configuration-for-node-projects?

This already runs only on pushes to the master, but we may expand it to other branches for alpha/beta and backporting.